tests/Parser/ParserAbstractTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Findologic\PlentymarketsTest\Parser;
+
+use Findologic\Plentymarkets\Parser\ParserAbstract;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class ParserAbstractTest extends TestCase
+{
+    public function correctDomainUrlProvider(): array
+    {
+        return [
+            ['www.example.com/lt', 'www.example.com'],
```

Comment:
What happens if we put in an URL that is no URL? Can we throw an exception and ensure that it is thrown if that's the case?